### PR TITLE
PAINT-169 Numbering style for bidirectional languages

### PR DESCRIPTION
**Pull request by ndrnour**
Numbering styles vary depending on the region. bidirectional languages numbering style may be implemented by either Hindi digits or Arabic digits.

### CODE REVIEW

File: Paintroid/src/androidTest/java/org/catrobat/paintroid/test/integration/ArabicNumberFormatTest.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+import android.widget.TextView;
+
+import com.robotium.solo.Solo;
+
+import org.catrobat.paintroid.MainActivity;
+import org.catrobat.paintroid.R;
+
+import java.util.Arrays;
+
+//tested on SAMSUNG Galaxy s5
```
Comment (cbenkovic) [inline Paintroid/src/androidTest/java/org/catrobat/paintroid/test/integration/ArabicNumberFormatTest.java]: please make sure the tests are running on all devices and remove this comment

File: Paintroid/src/androidTest/java/org/catrobat/paintroid/test/integration/ArabicNumberFormatTest.java [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+//make sure that your PhoneLanguage is Arabic
```
Comment (cbenkovic) [inline Paintroid/src/androidTest/java/org/catrobat/paintroid/test/integration/ArabicNumberFormatTest.java]: how to make this sure in automatic tests?

File: Paintroid/src/androidTest/java/org/catrobat/paintroid/test/integration/ArabicNumberFormatTest.java [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+public class ArabicNumberFormatTest extends ActivityInstrumentationTestCase2<MainActivity> {
+    public ArabicNumberFormatTest() {
```
Comment (cbenkovic) [inline Paintroid/src/androidTest/java/org/catrobat/paintroid/test/integration/ArabicNumberFormatTest.java]: please use tabs for the indents as in (most of) the other .java files

File: Paintroid/src/androidTest/java/org/catrobat/paintroid/test/integration/ArabicNumberFormatTest.java (hunk 1 above)
Comment (ndrnour) [inline Paintroid/src/androidTest/java/org/catrobat/paintroid/test/integration/ArabicNumberFormatTest.java]: done

File: Paintroid/src/androidTest/java/org/catrobat/paintroid/test/junit/stubs/CommandManagerStub.java [hunk 4]
```diff
@@ -89,5 +95,15 @@ public void addCommandToList(LayerCommand layerCommand, Command command){
 
 	}
```
Comment (ndrnour) [inline Paintroid/src/androidTest/java/org/catrobat/paintroid/test/junit/stubs/CommandManagerStub.java]: those missed Override Methods should be implemented I guess

File: Paintroid/src/androidTest/java/org/catrobat/paintroid/test/junit/stubs/CommandManagerStub.java [hunk 5]
```diff
@@ -67,6 +68,11 @@ public void resetAndClear(boolean clearLayerBitmapCommandsList) {
...
 
 	@Override
+	public LayerBitmapCommand getLayerBitmapCommand(LayerCommand layerCommand) {
+		return null;
+	}
+
```
Comment (ndrnour) [inline Paintroid/src/androidTest/java/org/catrobat/paintroid/test/junit/stubs/CommandManagerStub.java]: a missed Override Method should be implemented I guess

File: Paintroid/src/androidTest/java/org/catrobat/paintroid/test/junit/stubs/CommandManagerStub.java [hunk 6]
```diff
@@ -89,5 +95,15 @@ public void addCommandToList(LayerCommand layerCommand, Command command){
...
+	@Override
+	public void enableUndo(boolean enable) {
+
+	}
+
+	@Override
+	public void enableRedo(boolean enable) {
+
+	}
+
```
Comment (ndrnour) [inline Paintroid/src/androidTest/java/org/catrobat/paintroid/test/junit/stubs/CommandManagerStub.java]: those missed Override Methods should be implemented I guess

**Comment by ndrnour** [comment]
missed Override Methods made that!
have a look

**Pull request merged**